Python/ibmcloudsql/SQLQuery.py

```diff
@@ -471,6 +990,93 @@ def get_jobs(self):
             print("Job list retrieval failed with http code {}".format(response.status_code))
         return job_list_df
 
+    def get_jobs_matching(self, job_id_list, status="completed"):
```

Comment:
get_jobs_by_status

Reply:
Updated in commit `db65de9fadb7056698c5690333171be3bdb34130`